api/v1beta1/schedule_types.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	veleroapi "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+)
+
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// BackupScheduleSpec defines the desired state of BackupSchedule
+type BackupScheduleSpec struct {
+	// Schedule is a Cron expression defining when to run
+	// the Velero Backup
+	// +kubebuilder:validation:Required
+	VeleroSchedule string `json:"veleroSchedule"`
+	// TTL is a time.Duration-parseable string describing how long
+	// the Velero Backup should be retained for.
+	// +kubebuilder:validation:Required
+	VeleroTTL metav1.Duration `json:"veleroTtl,omitempty"`
+}
+
+// BackupScheduleStatus defines the observed state of BackupSchedule
+type BackupScheduleStatus struct {
+	// Phase shows the status for the backup operation
+	// +kubebuilder:validation:Optional
+	Phase StatusPhase `json:"phase"`
+	// Velero Schedule for backing up remote clusters
+	// +kubebuilder:validation:Optional
+	VeleroScheduleManagedClusters *veleroapi.Schedule `json:"veleroManagedClustersSchedule,omitempty"`
```

Comment:
I would prefer to be consistent with variable name and json tags... Either
`VeleroScheduleManagedClusters *veleroapi.Schedule `json:"veleroScheduleManagedClusters,omitempty"`
or 
`VeleroManagedClustersSchedule *veleroapi.Schedule `json:"veleroManagedClustersSchedule,omitempty"`